utils/common.sh

```diff
@@ -56,4 +56,16 @@ _open_url() {
   elif which open > /dev/null; then # macOS
     open "${url}"
   fi
-}
\ No newline at end of file
+}
+
+# generates a password that can be safely used in shells by exluding special characters
+# param 1: the requested length of the password, by default 64
+# param 2: additional characters to exclude
+_generate_shell_safe_password() {
+  local length="${1:-64}"
```

Comment:
I dont have much experience with default values in bash functions. I thought it makes sense to add the default 64, the last parameter is purely optional, so we can not move the length with default value to the end as otherwise the other parameter becomes mandatory if you want to change the length of the password.

We also can remove the default ofc.

Reply:
the length is now the last parameter